content/01.abstract.md

```diff
@@ -1,3 +1,18 @@
 ## Abstract {.page_break_before}
 
-This is a **rough draft** of a manscript on label function reuse for text mining heterogenous relationship from Pubmed Abstracts.
+Knowledge-bases are useful resources that aide in multiple processes such as network construction and analysis, providing researchers contextual information, and more.
+Majority of these knowledge-bases are populated through manual curation.
```

Comment:
I'm not entirely sure what you're going for here. It would be `A majority` or `Most` from a grammar point of view. My uncertainty is deeper though.

Do you mean that there are knowledge bases that are entirely automatically derived, but most are manually derived? Or do you mean that of knowledge bases, they include edges - some of which are automatically derived but the majority are manually curated?

Reply:
From my understanding, I was trying to say that most knowledge bases that include edges are populated via some form of manual curation.